### ISSUE TITLE
超参数解释问题

### ISSUE THREAD

**Issue by gebilxs** [TsingZ0/FedALA#4]
论文中的p和s是什么超参数呀？我只在代码中看到 cid: int,
                loss: nn.Module,
                train_data: List[Tuple], 
                batch_size: int, 
                rand_percent: int, 
                layer_idx: int = 0,
                eta: float = 1.0,
                device: str = 'cpu', 
                threshold: float = 0.01,
                num_pre_loss: int = 10) -> None: 这些参数

**Comment by TsingZ0** [comment]
layer_idx是p，rand_percent是s。
我重新修改一下代码
